Source/Lib/Codec/EbEncHandle.c

```diff
@@ -2980,7 +3002,8 @@ EB_API EB_ERRORTYPE EbH265EncStreamHeader(
         outputStreamBuffer->pBuffer,
         (EB_U32*) &(outputStreamBuffer->nFilledLen),
         (EB_U32*) &(outputStreamBuffer->nAllocLen),
-        encodeContextPtr);
+        encodeContextPtr,
+        9999);
```

Comment:
Can we define a macro for this value ? what does 999 refer to ?

Reply:
Thanks for Review comments, I have modified code and sent Patch.